ppgan/models/discriminators/dcdiscriminator.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
@LielinJiang  please see whether this discriminator  use reuse  https://github.com/PaddlePaddle/PaddleGAN/blob/master/ppgan/models/discriminators/nlayers.py

Reply:
yes, sure